client/blocks/importer/wordpress/upgrade-plan/index.tsx

```diff
@@ -2,34 +2,21 @@ import { recordTracksEvent } from '@automattic/calypso-analytics';
 import { isEnabled } from '@automattic/calypso-config';
 import { getPlan, PLAN_BUSINESS } from '@automattic/calypso-products';
 import { Button } from '@automattic/components';
-import { SiteDetails } from '@automattic/data-stores';
+import { Plans } from '@automattic/data-stores';
 import { useHasEnTranslation, useIsEnglishLocale } from '@automattic/i18n-utils';
 import { Title, SubTitle, NextButton } from '@automattic/onboarding';
 import { useTranslate } from 'i18n-calypso';
 import React, { useEffect } from 'react';
 import useCheckEligibilityMigrationTrialPlan from 'calypso/data/plans/use-check-eligibility-migration-trial-plan';
 import PlanNoticeCreditUpgrade from 'calypso/my-sites/plans-features-main/components/plan-notice-credit-update';
+import { useUpgradePlanHostingDetailsList } from './hooks/use-get-upgrade-plan-hosting-details-list';
+import { Skeleton } from './skeleton';
 import UpgradePlanDetails from './upgrade-plan-details';
-import type { PlanSlug } from '@automattic/calypso-products';
-
 import './style.scss';
+import withMigrationSticker from './with-migration-sticker';
+import type { UpgradePlanProps } from './types';
 
-interface Props {
-	site: SiteDetails;
-	isBusy: boolean;
-	ctaText: string;
-	subTitleText?: string;
-	hideTitleAndSubTitle?: boolean;
-	onFreeTrialClick?: () => void;
-	navigateToVerifyEmailStep: () => void;
-	onCtaClick: () => void;
-	onContentOnlyClick?: () => void;
-	trackingEventsProps?: Record< string, unknown >;
-	hideFreeMigrationTrialForNonVerifiedEmail?: boolean;
-	visiblePlan?: PlanSlug;
-}
-
-export const UpgradePlan: React.FunctionComponent< Props > = ( props: Props ) => {
+export const UpgradePlanUnwrapped: React.FunctionComponent< UpgradePlanProps > = ( props ) => {
```

Comment:
Nice name solution you used here!

Nit: I'd suggest to use the "Unwrapped" as a prefix, just to use the same standard we used in the HOC (`WrappedComponent`).

Reply:
Updated here: https://github.com/Automattic/wp-calypso/pull/92201/commits/e4b21382979fbc2d9898892ce5eda5ccaee6f58e